src/app/view/components/_common/dot-autocomplete-tags/dot-autocomplete-tags.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DotTagsService } from '@services/dot-tags/dot-tags.service';
+import { DotTag } from '@models/dot-tag';
+import { filter, mergeMap, toArray } from 'rxjs/operators';
+
+/**
+ * The DotAutocompleteTagsComponent provide a dropdown to select tags,
+ * the output is an array of strings with the labels.
+ * @export
+ * @class DotAutocompleteTagsComponent
+ */
+@Component({
+    selector: 'dot-autocomplete-tags',
+    templateUrl: './dot-autocomplete-tags.component.html',
+    styleUrls: ['./dot-autocomplete-tags.component.css']
+})
+export class DotAutocompleteTagsComponent implements OnInit {
+    @Input() inputId: string;
+    @Input() value: string[] = [];
+    @Input() placeholder: string;
+
+    @Output() onChange = new EventEmitter<any>();
+
+    filteredOptions: DotTag[];
+
+    constructor(private dotTagsService: DotTagsService) {}
+
+    ngOnInit() {
+        this.filterTags();
+        this.value = this.value === null ? [] : this.value;
```

Comment:
I which case can `this.value` be `null`?

Reply:
in case when we need to pre-load tags, but comes empty/null

Reply:
not needed anymore